Felienne/hedy
# [FEATURE] Enables invite to class by username

**Description**
In this PR we implement a new feature, enabling teachers to invite students by their username. Currently teachers are only able to add students to their classes by copying and sharing the "join link". As discussed in #1735 some school don't have mail accounts or an easy way of sharing this link.

We add a new button to the class overview for "add student". When pressing this button the prompt modal will pop-up and the teacher is able to enter an username. If this is a valid account (and not already in their class) the username will be stored in a dedicated `invitations` db table. When a user logs in this table is checked if there are any pending records for the user. If so: they are shown in their profile.

**Fix for**
This PR fixes #1735

**How to test**
Make sure you are a teacher and navigate to the overview page of any of your classes. Notice the new "add student" button and verify that you can add another (existing!) account that is not already in your class. When loggin in to this account the profile should contain a new message with the link to join the class.
